### PR TITLE
MovieLens Similar Items RecList

### PR DESCRIPTION
**Pull request by briankosw**

### CODE REVIEW

File: reclist/recommenders/prod2vec.py [hunk 1]
```diff
@@ -52,3 +52,31 @@ def get_vector(self, product_sku):
...
+
+class MovieLensP2VRecModel(RecModel):
+
+    model_name = "prod2vec"
+
+    def __init__(self, **kwargs):
+        super().__init__(**kwargs)
+
+    def train(self, movies):
+        self._model = train_embeddings(movies)
```
Comment (vinid) [inline reclist/recommenders/prod2vec.py]: Is there a reason why here we don't have the "movie_id" field but we have it in

`predictions.append([{"movie_id": elem} for elem in nn_products])`

Is there any way we can keep it consistent?

File: examples/movielens_similar_rec.py [hunk 2]
```diff
@@ -0,0 +1,38 @@
+"""
+    This is a sample script showing how to get started with reclist. In particular,
+    we are targeting a "similar items" use cases, such as for example a item recommender:
+    if a shopper is browsing a product listing and clicked on item X, what other item
+    might they be interested in that is similar in type to item X?
+
+    We train a simple, yet effective prod2vec baseline model (https://arxiv.org/abs/2007.14906),
+    re-using for convenience a "training embedding" function already implemented by recsys. The script
+    show how easy it is to run behavioral tests on a target dataset, in this case a wrapper around a large
+    e-commerce dataset (the Coveo Data Challenge dataset: https://github.com/coveooss/SIGIR-ecom-data-challenge).
```
Comment (vinid) [inline examples/movielens_similar_rec.py]: Movielens?

**Comment by vinid** [comment]
can you also add this to the tests? (just copy paste the example for now)

File: reclist/recommenders/prod2vec.py (hunk 1 above)
Comment (briankosw) [reply reclist/recommenders/prod2vec.py]: Yeah I had mismatching datasets which resulted in mismatching train and predict. I'll fix that right now!

**Comment by briankosw** [comment]
@vinid I added a new metric for calculating hit rate distribution across different movie ratings. I'd love any feedback on that :)

**Comment by vinid** [comment]
awesome work @briankosw! I'll start merging this so we can start building some notebooks to play with it :)

**Comment by vinid** [comment]
I guess we can work on master in the meantime

**Pull request merged**